gtd-timer/Constants/Constants.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace gtdtimer.Constant
+{
+    public class Constants
+    {
+        public const string SecretKey = "MyKey";
+        public const string RequiredUser = "Username is required";
+        public const int Status401Unauthorized = 401;

```

Comment:
no need for this - there is standard HttpStatusCode enum: https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netframework-4.7.2